manifests/concourse-manifest/spec/manifest_generation_spec.rb

```diff
@@ -35,7 +35,7 @@ def merge_fixtures(fixtures)
 
   it "gets values from secrets" do
     expect(
-      atc_job.fetch("properties").fetch("basic_auth_password")
+      atc_job.fetch("properties").fetch("add_local_users")[0].sub(/^[^:]+:/, '')
```

Comment:
I wonder whether this would be clearer using split - would save having to mentally parse a regexp:
```ruby
atc_job.fetch("properties").fetch("add_local_users")[0].split(':', 2)[1]
```

Reply:
Passwords /might/ contain colons. Not with out current secret generator, but assuming that that code is driving this value seems like unnecessary coupling.

I feel like, for safety, we'd need to pop the first element after the split, then join the remaining fields with `:`s. Is that simpler than the current regex, and perhaps adding a comment?

Reply:
I'd accounted for that - the `2` param to `split` constrains it to at most 2 results:
```ruby
"foo:bar:baz".split(':',2)[1] # => "bar:baz"
```